### PR TITLE
🎨 fix todos

### PR DESCRIPTION
**Pull request by j-vp**

### CODE REVIEW

File: src/asset-modules/AbstractDerivedAssetModule.sol [hunk 1]
```diff
@@ -482,7 +483,7 @@ abstract contract DerivedAssetModule is AssetModule {
 
             // Cache and update lastUsdExposureAsset.
             uint256 lastUsdExposureAsset = lastExposuresAsset[creditor][assetKey].lastUsdExposureAsset;
-            lastExposuresAsset[creditor][assetKey].lastUsdExposureAsset = uint112(usdExposureAsset); // ToDo safecast.
+            lastExposuresAsset[creditor][assetKey].lastUsdExposureAsset = uint112(usdExposureAsset);
```
Comment (Thomas-Smets) [inline src/asset-modules/AbstractDerivedAssetModule.sol:486]: // If usdExposureAsset is bigger than uint112, then safecast on usdExposureProtocol below will revert.

File: src/asset-modules/AbstractDerivedAssetModule.sol [hunk 2]
```diff
@@ -556,6 +557,7 @@ abstract contract DerivedAssetModule is AssetModule {
 
         // Cache and update lastUsdExposureAsset.
         uint256 lastUsdExposureAsset = lastExposuresAsset[creditor][assetKey].lastUsdExposureAsset;
+        // If usdExposureAsset is bigger than uint112, then safecast on L577 will revert.
```
Comment (Thomas-Smets) [inline src/asset-modules/AbstractDerivedAssetModule.sol:560]: // If usdExposureAsset is bigger than uint112, then safecast on usdExposureProtocol below will revert.

**Review by Thomas-Smets — APPROVED** [review]
gtg after comment

File: src/asset-modules/AbstractDerivedAssetModule.sol (hunk 1 above)
Comment (j-vp) [reply src/asset-modules/AbstractDerivedAssetModule.sol:487]: changed safecast below to point to the usdExposure check with the maxexposure

**Pull request merged**